app/pages/neworg/findorg/controller.js

```diff
@@ -1,3 +1,17 @@
 import { getContext } from './contextCreator';
+import { getData } from '../../../apiProvider';
 
-export const getFindOrgContext = async ({ odsCode }) => getContext({ odsCode });
+export const getFindOrgContext = params => getContext(params);
+
+export const getFindOrgByOds = async ({ odsCode, accessToken }) => {
+  try {
+    await getData({ endpointLocator: 'getOrgByOdsCode', options: { odsCode }, accessToken });
+
+    return { success: true };
+  } catch (err) {
+    if (err.status === 404 || err.status === 406) {
+      return { success: false, errorStatus: err.status };
```

Comment:
Does the api return a key in the response of the body? I don't think we should be leaking these http status codes back up. But should return the errorKey instead perhaps

Reply:
no - just status and message

Reply:
oh that's bit pants. I don't like these statusCodes being leaked up. If we get anymore variations of errors we can revisit this then.

Reply:
I don't really see a problem with it - at least it is short and reduces risk of mutation

Reply:
it just something that the api returns that means something that creeps up the stack which the also means something. Then if this changes we have to change it a few places.
If for example the Api decides to return a 400 in the future and a key in the body we would have to change it in a few places too. 

Ideally I think the apiProvider should really be abstracting this out for us. One for tech debt